packages/flutter/lib/src/painting/image_stream.dart

```diff
@@ -489,6 +491,9 @@ abstract class ImageStreamCompleter with Diagnosticable {
   /// if all [keepAlive] handles get disposed.
   bool _hadAtLeastOneListener = false;
 
+  /// Whether the listeners should be called synchronously
+  bool _synchronousCall = true;
```

Comment:
In case it was buried in previous commit.

maybe renaming this to _addingInitialListener to be more self-explanatory. We may also want to add some comment on this variable to explain why this is here and what happen if this is set to true vs false

Reply:
I have renamed it to _hasInitialListeners and provided some documentation.